source/changes-to-govuk-frontend-v5/index.html.md.erb

```diff
@@ -0,0 +1,59 @@
+---
+title: Changes to GOV.UK Frontend v5.0.0
+weight: 41
```

Comment:
@claireashworth Order wise, this will place it above the Browser Support link. Just confirming that's the expected place, though @domoscargin might already have checked with you 😊 

<img width="1792" alt="Screenshot 2023-10-26 at 14 32 58" src="https://github.com/alphagov/govuk-frontend-docs/assets/396367/546b2727-2ea0-4830-8592-418890576c54">

Reply:
yes please